lua/astrocommunity/bars-and-lines/heirline-mode-text-statusline/init.lua

```diff
@@ -1,7 +1,7 @@
 return {
   "rebelot/heirline.nvim",
   opts = function(_, opts)
-    local status = require "astronvim.utils.status"
+    local status = require "astrocore.status"
```

Comment:
Astroui has status now.
```suggestion
    local status = require "astroui.status"
```